samples/nrf9160/aws_fota/src/certificates.h

```diff
@@ -11,7 +11,7 @@
 
 #define CLOUD_CLIENT_PUBLIC_CERTIFICATE \
 	"-----BEGIN CERTIFICATE-----\n" \
-	"AWS IoT certificate public key\n"\
+	"AWS IoT certificate\n"\
```

Comment:
I'd make it `AWS IoT client certificate` to be specific here.

Reply:
Added